main.tf

```diff
@@ -0,0 +1,98 @@
+locals {
+
+  create_deployment_id = var.deployment_id != "" ? 0 : 1
+  # Common tags to be assigned to all resources
+  persistent_tags = {
+    purpose = "automation"
+    environment   = "ansible-automation-platform"
+    deployment = "aap-infrastructure-${var.deployment_id}"
+  }
+}
+
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "=3.83.0"
+    }
+  }
```

Comment:
```suggestion
  }
  required_version = ">= 1.4.0"
```

Reply:
Shouldn't this be a newer version?  I currently have 1.6.4 but I see aws_terraform_deployment has 1.6.5

Reply:
latest terraform version is `1.6.5` but this project can work with any version greater than 1.4.0
I haven't tested it `1.4.0` I have `1.5.4`

Reply:
I would go with what was tested as the minimum. If you tested with 1.5.4, let's set it to that.